examples/pytorch/BertNewsClassification/README.md

```diff
@@ -41,35 +24,45 @@ If you have the required modules for the file and would like to skip the creatio
 mlflow run . --no-conda
 ```
 
-### Example with custom input
+### Viewing results in the MLflow UI
+Once the code is finished executing, you can view the run's metrics, parameters, and details by running the command
+
+```
+mlflow ui
+```
+
+and navigating to [http://localhost:5000](http://localhost:5000).
+
+For more details on MLflow tracking, see [the docs](https://www.mlflow.org/docs/latest/tracking.html#mlflow-tracking).
+
+### Passing custom training parameters
 
-Following are the parameters which can be overridden by passing values in command line argument.
+The parameters below can be overridden via the command line:
+
+1. ``max_epochs`` - Number of epochs to train model. Training can be interrupted early via Ctrl+C
+2. ``gpus`` - Number of GPUs
+3. ``accelerator`` - [Accelerator backend](https://pytorch-lightning.readthedocs.io/en/latest/trainer.html#trainer-flags)
+   (e.g. "ddp" for the Distributed Data Parallel backend) to use for training. By default, no accelerator is used.
+4. ``batch-size`` - Input batch size for training
+5. ``num-workers`` - Number of worker threads to load training data
+6. ``lr`` - Learning rate
 
-1. Number of epochs - max_epochs
-2. Number of gpus - gpus
-3. Backend in case of gpus environment - accelerator
-4. Batch size to process - batch-size
-5. Number of workers to process input - num-workers
-6. Learning rate - lr
 
 For example:
 ```
+mlflow run . -P epochs=5 -P gpus=1 -P batch_size=32 -P num_workers=2 -P learning_rate=0.01 -P accelerator="ddp"
```

Comment:
@karthik-77 I added an example here of how to run the training script as an MLflow project with custom parameters, for consistency / logical flow from the previous section, which explains how to run training script as a project. Could we add a similar section to the other READMEs as well?